Detections/ZoomLogs/JoiningMeetingFromAnotherTimeZone.yaml

```diff
@@ -0,0 +1,40 @@
+id: 58fc0170-0877-4ea8-a9ff-d805e361cfae
+name: User joining Zoom meeting from suspicious timezone
+description: |
+  'The alert shows users that join a Zoom meeting from a time zone other than the one the meeting was created in.
+  You can also whitelist known good time zones in the tz_whitelist value using the tz database name format https://en.wikipedia.org/wiki/List_of_tz_database_time_zones'
+severity: Low
+requiredDataConnectors:
+  - connectorId: CustomConnector
+    dataTypes:
+      - ZoomLogs
+queryFrequency: 6h
+queryPeriod: 6h
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - Initial Access
+relevantTechniques:
+  - T1078
+query: |
+
+  let schedule_lookback = 30d; 
+  let join_lookback = 6h; 
+  // If you want to whitelist specific timezones include them in a list here
+  let tz_whitelist = dynamic([]);
+  let meetings = ( 
+  ZoomLogs 
+  | where TimeGenerated > ago(schedule_lookback) 
+  | where Event == "meeting.created" 
+  | extend MeetingId = tostring(parse_json(MeetingEvents).MeetingId)  
+  | extend SchedTimezone = tostring(parse_json(MeetingEvents).Timezone)); 
+  ZoomLogs 
+  | where TimeGenerated > ago(join_lookback) 
+  | where Event == "meeting.participant_joined" 
+  | extend JoinedTimeZone = tostring(parse_json(MeetingEvents).Timezone) 
+  | extend MeetingName = tostring(parse_json(MeetingEvents).MeetingName) 
+  | extend MeetingId = tostring(parse_json(MeetingEvents).MeetingId) 
+  | where JoinedTimeZone !in (tz_whitelist)
+  | join (meetings) on MeetingId 
+  | where SchedTimezone != JoinedTimeZone 
+  | project TimeGenerated, MeetingName, JoiningUser=payload_object_participant_user_name_s, JoinedTimeZone, SchedTimezone, MeetingScheduler=User1 
```

Comment:
| extend timestamp = TimeGenerated, AccountCustomEntity = JoiningUser <-- not sure you want JoiningUser or MeetingScheduler here?